cypress/e2e/models/administration/source-platform/source-platform.ts

```diff
@@ -143,4 +144,32 @@ export class SourcePlatform {
             submitForm();
         }
     }
+
+    discover(name: string, space: string, cancel = false): void {
+        SourcePlatform.open();
+        clickItemInKebabMenu(this.name, "Discover applications");
```

Comment:
we are taking name as an argument then why are we using this.name ? 
If both names are different can we have a different identifier for the argument.